src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const male = people.filter(person => person.sex === 'm');
+  const peopleOfCentury = male.filter(person =>
+    Math.ceil(person.died / 100) === century);
```

Comment:
no need to calculate it if century isn't passed

Reply:
I didn't understand it. Could you explain this more detail? how can I improve this code?